src/components/common/Banner.js

```diff
@@ -18,20 +18,36 @@ function bannerContent(title, content) {
 export default function Banner() {
   const path = window.location.pathname;
   switch (path) {
-    case '/parent' || '/instructor':
+    case '/instructor':
       return bannerContent('Dashboard', 'welcome back!');
-    case '/student':
-      return bannerContent('Student Dashboard');
-    case '/parent-booking':
-      return bannerContent('Courses', "view your children's reserved courses");
     case '/instructor-booking':
       return bannerContent('Apply to Course', 'Select Courses');
     case '/instructor-add-course':
       return bannerContent('Create', 'your own course');
-    case '/family':
-      return bannerContent('CHOOSE A PROFILE');
     case '/instructor-news-feed' || '/parent-news-feed':
       return bannerContent('News Feed');
+    case '/parent':
+      return bannerContent('Mission Control', '');
+    case '/parent-progress':
+      return bannerContent('Progress', "view your children's progress");
+    case '/parent/calendar':
+      return bannerContent('Calendar', '');
+    case '/parent/booking':
+      return bannerContent('Courses', '');
+    case '/parent/newsfeed':
+      return bannerContent('News Feed', '');
+    case '/parent/family':
+      return bannerContent('CHOOSE A PROFILE');
+    case '/parent/cart':
+      return bannerContent('Cart');
+    case '/parent/messages':
+      return bannerContent('Messages');
+    case '/parent/tasks':
+      return bannerContent('TASKS');
```

Comment:
Looks clean, is there a reason why the bannerContent text is in caps for some of these? Like, TASKS, RESOURCES, CHOOSE A PROFILE ?